test/hotspot/jtreg/serviceability/sa/ClhsdbSymbol.java

```diff
@@ -0,0 +1,113 @@
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import jdk.test.lib.apps.LingeredApp;
+import jtreg.SkippedException;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @test
+ * @bug 8261095
+ * @summary Test the clhsdb 'symbol' command on live process
+ * @requires vm.hasSA
+ * @library /test/lib
+ * @run main/othervm ClhsdbSymbol
+ */
+
+public class ClhsdbSymbol {
+
+    public static void main(String[] args) throws Exception {
+        System.out.println("Starting the ClhsdbSymbol test");
+        LingeredApp theApp = null;
+
+        try {
+            List<String> cmds = null;
+            String cmdStr = null;
+            Map<String, List<String>> expStrMap = null;
+            ClhsdbLauncher test = new ClhsdbLauncher();
+            theApp = LingeredApp.startApp();
+            System.out.println("Started LingeredApp with pid " + theApp.getPid());
+
+            //Use command "class java.lang.Thread" to get the address of the InstanceKlass for java.lang.Thread
+            cmdStr = "class java.lang.Thread";
+            cmds = List.of(cmdStr);
+            expStrMap = new HashMap<>();
+            expStrMap.put(cmdStr, List.of("java/lang/Thread"));
+            String classOutput = test.run(theApp.getPid(), cmds, expStrMap, null);
+            String threadAddress = null;
+            String[] parts = classOutput.split("\n");
+
+            //extract thread address from the output line similar to "java/lang/Thread @0x000000080001d940"
+            for (String part : parts) {
+                if (part.startsWith("java/lang/Thread")) {
+                    String[] addresses = part.split(" @");
+                    threadAddress = addresses[1];
+                    break;
+                }
+            }
+            if (threadAddress == null) {
+                throw new RuntimeException("Cannot find address of the InstanceKlass for java.lang.Thread in output");
+            }
+
+            //Use "inspect" on the thread address we extracted in previous step
+            cmdStr = "inspect " + threadAddress;
+            cmds = List.of(cmdStr);
+            expStrMap = new HashMap<>();
+            expStrMap.put(cmdStr, List.of("Symbol"));
+            String inspectOutput = test.run(theApp.getPid(), cmds, expStrMap, null);
+            String symbolAddress = null;
+            parts = inspectOutput.split("\n");
+
+            //extract address comes along with Symbol instance, following is corresponding sample output line
+            //Symbol* Klass::_name: Symbol @ 0x0000000800471120
```

Comment:
// The inspect command output will have one line of output that looks like the following.
//   Symbol* Klass::_name: Symbol @ 0x0000000800471120
// Extract the Symbol address from it.